src/ensembl/stories/shared-components/slide-toggle/SlideToggle.stories.tsx

```diff
@@ -8,7 +8,7 @@ storiesOf('Components|Shared Components/SlideToggle', module).add(
   () => {
     return (
       <div>
-        <SlideToggle isOn={false} onChange={() => console.log('change')} />
+        <SlideToggle isOn={false} onChange={() => true} />
```

Comment:
() => {} ?
or import noop from 'lodash/noop'?

Reply:
used {}

Reply:
This has been changed to `noop` by @jyothishnt